static/js/components/dashboard/CourseAction.js

```diff
@@ -93,7 +93,7 @@ export default class CourseAction extends React.Component {
 
     return (
       <SpinnerButton
-        className="dashboard-button"
+        className="dashboard-button pay-button"
```

Comment:
are these changes related to rebasing over another change? Just wondering why there are changes to `CourseAction` in this pr.